packages/w3up-client/README.md

```diff
@@ -153,19 +136,61 @@ try {
 }
 ```
 
-By default, calling `registerSpace` registers the Space with web3.storage w3up. You can pass the optional `provider` param to register with NFT.Storage w3up instead.
+If you created the Space in w3cli using `w3 space create`, you can register the space by calling [`w3 space register`](https://github.com/web3-storage/w3cli#w3-space-register).
+
+#### Delegating from Space to Agent
+
+In order to store data with w3up, your Agent will need a delegation from a Space. The easiest way to do this is by authorizing your Agent. This allows you to claim delegations to the Space that is authorized to the _same_ email account. Authorization needs to happen only once per agent.
 
 ```js
-try {
-  await client.registerSpace('zaphod@beeblebrox.galaxy', { provider: 'did:web:nft.storage' })
-} catch (err) {
-  console.error('registration failed: ', err)
+await client.authorize('zaphod@beeblebrox.galaxy')
+```
+
+Calling `authorize` will cause an email to be sent to the given address. Once a user clicks the confirmation link in the email, the `authorize` method will resolve. Make sure to check for errors, as `authorize` will fail if the email is not confirmed within the expiration timeout.
+
+If this is not the first time you authorized an agent with your email, then you'll want to claim any spaces and delegations you have on your other agent(s):
```

Comment:
fwiw this will be run automatically when they call `client.authorize` - no need to call it separately

Reply:
just clarifying, claim is only useful if your agent is already authorized but there's a new space? (this was leftover from the README before)

Reply:
yea I think that's right - if another agent creates a space and registers it to your account, and you've already authorized, you'd probably want to `claim()` - can't think of other situations where you'd use that currently